concordium-consensus/src/Concordium/ProtocolUpdate/P7.hs

```diff
@@ -8,29 +8,33 @@ module Concordium.ProtocolUpdate.P7 (
     updateNextProtocolVersion,
 ) where
 
-import Control.Monad.State
+import Control.Monad.State hiding (get)
 import qualified Data.HashMap.Strict as HM
 import Data.Serialize
 
 import qualified Concordium.Crypto.SHA256 as SHA256
 import Concordium.Types
 import Concordium.Types.Updates
 
+import qualified Concordium.Genesis.Data.P8 as P8
 import Concordium.GlobalState.BlockState
 import qualified Concordium.GlobalState.Persistent.BlockState as PBS
 import Concordium.GlobalState.Types
 import qualified Concordium.GlobalState.Types as GSTypes
 import Concordium.KonsensusV1.TreeState.Implementation
 import Concordium.KonsensusV1.TreeState.Types
+import qualified Concordium.ProtocolUpdate.P7.ProtocolP8 as ProtocolP8
 import qualified Concordium.ProtocolUpdate.P7.Reboot as Reboot
```

Comment:
I think this should be P8.Reboot.

Reply:
In this file we specify all the protocol updates that are possible from P7. Concretely we specify two of such updates, namely a P7 -> P7 update (the `Reboot` update) and a P7 -> P8 update (the update that introduces P8).

I there believe that we should indeed use ``P7.Reboot` and not `P8.Reboot` as you suggest.

Reply:
You're right, I sought I was looking at P8.hs.